index.bs

```diff
@@ -0,0 +1,505 @@
+<pre class=metadata>
+Title: Writable Files
+Shortname: WritableFiles
+Abstract: This document defines a web platform API that lets websites gain write access to the
+  native file system. It builds on [[FILE-API]], but adds lots of new functionality on top.
+Status: ED
+ED: https://wicg.github.io/writable-files/
+Level: 1
+Editor: Marijn Kruisselbrink, Google, mek@chromium.org, w3cid 72440
+Group: WICG
+Repository: wicg/writable-files
+Indent: 2
+Complain About: accidental-2119 yes, missing-example-ids yes
+Markup Shorthands: css no, markdown yes
+</pre>
+
+<pre class=link-defaults>
+spec:fetch; type:interface; text:ReadableStream
+spec:infra; type:dfn; for:/; text:set
+</pre>
+
+<pre class=anchors>
+</pre>
+
+<style>
+dl.domintro dt {

```

Comment:
I know you're replicating what other specs (mine?) do, but if we changed these selectors from `dl.domintro` to just `.domintro` then we could use `<div class="note domintro">` below and avoid the explicit `<dl>` tag.

Reply:
Ah yes, that's nicer. Done.